internal/server/server.go

```diff
@@ -1160,7 +1160,31 @@ func (server *Server) command(msg *Message, client *Client, ts *txn.Status) (
 	case "config get":
 		res, err = server.cmdConfigGet(msg)
 	case "config set":
-		res, err = server.cmdConfigSet(msg)
+		var configName string
```

Comment:
Do we hold the write lock here? I suspect not given the locking of config.mu. That means that updating the tree values needs to be synchronized in some other way I think.